src/Entries/Collection.php

```diff
@@ -710,11 +710,31 @@ public function previewTargets($targets = null)
                     $targets = $this->defaultPreviewTargets();
                 }
 
-                return collect($targets);
+                return collect($targets)->merge($this->extraPreviewTargets());
             })
             ->args(func_get_args());
     }
 
+    public function extraPreviewTargets($targets = null)
+    {
+        if (! $targets) {
+            return \Statamic\Facades\Collection::extraPreviewTargets($this->handle);
+        }
+
+        \Statamic\Facades\Collection::addExtraPreviewTargets($this->handle, $targets);
+
+        return $this;
+    }
+
+    public function previewTargetsWithoutExtra()
+    {
+        return $this->previewTargets()->filter(function ($target) {
+            return $this->extraPreviewTargets()->doesntContain(function ($extraTarget) use ($target) {
+                return $target['format'] === $extraTarget['format'];
+            });
+        });
+    }
```

Comment:
This negative name feels a bit strange.

Could we instead have a `basePreviewTargets()` that just gives you what `previewTargets()` _used to_ give you?

Then have `previewTargets()` merge `basePreviewTargets()` and `extraPreviewTargets()`?